troposphere/static/js/actions/instance/report.js

```diff
@@ -4,6 +4,7 @@ define(function (require) {
   var stores = require('stores'),
     globals = require('globals'),
     $ = require('jquery'),
+    _ = require('underscore'),
```

Comment:
I'm pretty shocked that we didn't see these errors popping up on issue-report attempts sooner. 

I guess this is an example of collecting errors :horse: